skyplane/api/tracker.py

```diff
@@ -264,10 +264,11 @@ def monitor_single_dst_helper(dst_region):
     def monitor_transfer(pd, self, region_tag):
         """Monitor the tranfer by copying remote gateway logs and show transfer stats by hooks"""
         # todo implement transfer monitoring to update job_complete_chunk_ids and job_pending_chunk_ids while the transfer is in progress
-        region_sinks = self.dataplane.topology.sink_instances()
-        sinks = region_sinks[region_tag]
-        # for region_tag, sink_gateways in self.dataplane.topology.sink_gateways().items():
-        # sink_regions = set([sink.region for sink in sinks])
+
+        # regions that are sinks for specific region tag
+        # TODO: should eventualy map bucket to list of instances
+        sinks = [n for nodes in self.dataplane.topology.sink_instances(region_tag).values() for n in nodes]
```

Comment:
I'm a bit confused as to what this is doing?

Reply:
This is grabbing all the instances that are writing to a bucket in `region_tag` - previously the sink instances were defined as the instances in the sink region, but that doesn't hold for one-sided transfers